zone/lua_mob.cpp

```diff
@@ -3303,6 +3304,20 @@ std::string Lua_Mob::GetDeityName()
 	return EQ::deity::GetDeityName(static_cast<EQ::deity::DeityType>(self->GetDeity()));
 }
 
+Lua_Buffs Lua_Mob::GetBuffs() {
+	Lua_Safe_Call_Class(Lua_Buffs);
+	Lua_Buffs ret;
+
+	const auto& buffs = self->GetBuffs();
+
+	for (int slot_id = 0; slot_id < self->GetMaxBuffSlots(); slot_id++) {
+		Lua_Buff e(&buffs[slot_id]);
+		ret.entries.push_back(e);
```

Comment:
It would be better to just return a lua table from this function instead of using the luabind iterators

Reply:
I was curious about that, I was just following the format the other stuff had done. Certainly simplifies it to not have the entries part.